src/actions/go-to.ts

```diff
@@ -0,0 +1,22 @@
+import 'colors';
+import { Browser } from 'puppeteer';
+import Action from '../action.i';
+
+/**
+ * Opens a new window for a given browser and navigates to a specified url.
+ */
+
+export default class GoTo implements Action {
+  constructor() {}
+
+  // @ts-ignore
+  async execute(title: string, args: string[], outs: any[], context: any): Promise<any> {
```

Comment:
Return type should be `Promise<string[]>`

Reply:
Done